docs/docker.md

```diff
@@ -88,3 +88,53 @@ docker run -d -p 3013:3013 --hostname node0 \
 ```
 
 **Note: make sure `hostname` option is set when reusing the mnesia data directory**
+
+# Localnet
```

Comment:
This seems relevant enough to be mentioned in the release notes. Would you mind adding a note in the [release notes first paragraph](https://github.com/aeternity/epoch/blob/bc08fc66a61a6cd70f8b926107d5e5f5890dc78a/RELEASE-NOTES.md#about-this-release) deleting all the other stuff - as this is the first user-visible improvement in this release cycle?

Reply:
I added a row in c928c53 and removed old stuff, however I could come in with something good for the very first removed it until the actual release.